friends/src/components/friends.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Link, NavLink } from 'react-router-dom';
+
+function friends(props){
+
+    return (
+    <div>  
+        {/* {button we use to show our add new friend form; navlink is used to make button dissappear if form is rendered} */}
+        <div className='addFriend-btn-container'>
+            <NavLink to='/friends/addFriend' className='normal' activeClassName='active' ><button onClick={() => props.addBtnClear()}>Add New Friend</button></NavLink>
```

Comment:
This is a very long line. Try to break it up to make it more readable:
```javascript
<NavLink to='/friends/addFriend' className='normal' activeClassName='active'>
  <button onClick={() => props.addBtnClear()}>Add New Friend</button>
</NavLink>
```
Also, your onClick is a callback. Why? Is it that the `addBtnClear` isn't an arrow function? If so, then make it an arrow function where it's defined and then remove the extra step. If not, then go ahead and remove the extra step:
```javascript
<NavLink to='/friends/addFriend' className='normal' activeClassName='active'>
  <button onClick={props.addBtnClear}>Add New Friend</button>
</NavLink>
```